content/page/tea.md

```diff
@@ -6,10 +6,10 @@ toc: false
 draft: false
 menu: "sidebar"
 goimport: "code.gitea.io/tea git https://github.com/go-gitea/tea"
```

Comment:
missing change?

Reply:
yes, thank you for noticing. I've updated PR 😄